.buildkite/pipelines/on_merge_unsupported_ftrs.yml

```diff
@@ -127,3 +127,45 @@ steps:
           limit: 3
         - exit_status: '*'
           limit: 1
+
+  - command: .buildkite/scripts/steps/functional/security_serverless.sh
```

Comment:
@patrykkopycinski FYI.  These are migrated from the serverless pipeline, which was temporarily intended to run as a non-blocking check on-merge.  If there's a consensus that these are stable we can move these over too.

Reply:
I think most of the unstable tests were skipped in #162673 so it should be ok, but I will let Patryk decide.